### ISSUE TITLE
Add option under new game to use same settings as last/current game

### ISSUE THREAD

**Issue by vertigo220** [lucasnlm/antimine-android#498]
**Is your feature request related to a problem? Please describe.**
When I lose or just want to start a new game, I'd like to be able to do new game (which makes sense as the way to do so) without having to select the game parameters. I'd like to just be able to do new game > current settings or last game settings or something like that. Currently, to do that, assuming you didn't select the new game option in the popup when the game was lost, you have to continue the game and purposely detonate a mine to get the option to start a new game, which feels very clumsy.

**Describe the solution you'd like**
Add another option under new game in the main menu.

**Describe alternatives you've considered**
Add a new game button at the top on the game screen (with confirmation to prevent accidentally starting a new game in the middle of one) like clicking the smiley face at the top of the board in old-school minesweeper.

**Additional context**
N/A

**Comment by lucasnlm** [comment]
Makes sense. Ideally, `Continue` button is for that. If you have a game in progress, it will continue. If you have lost the last game, it will allow you to tap on restart.
But if you just want to ignore the current in progress game, there's no way.

I will check what I can do to solve that.

**Comment by vertigo220** [comment]
The problem is when I lose, then tap continue to go back to the game I just lost to examine it to see why I lost (related to #495), it continues on as if the game is still going. The timer continues to count up and the restart button turns back into the hint button. So there's no way to directly start a new game there, and going back to the main menu and trying to start a new game that way is harder than it should be since you have to reselect all the options again. The only way to "easily" start a new game with the same settings is to tap around until hitting a mine to "lose" again and this time select new game. So I see this as two separate problems:

1. When losing a game, it should be over and stay over, not continue when selecting continue, so that the restart button is available
2. The new game option in the main menu should have an option to start a new game with the last used settings (useful for coming back to the game)

As for ignoring the current progress and restarting, not sure why there would be no way. It could just always show the restart button and simply ask for confirmation if a game is currently going.
